examples/docs/src/config.ts

```diff
@@ -1,10 +1,31 @@
-export const sidebar = [
-  {
-    text: 'Introduction',
-    link: '', // No leading slash needed, so this links to the homepage
-    children: [
-      { text: 'Getting Started', link: 'getting-started' },
-      { text: 'Example', link: 'example' },
-    ],
+export const SIDEBAR = {
+  en: [
+    { text: 'Introduction', link: 'introduction', header: true },
```

Comment:
no introduction page anymore! sidebar headers are not clickable in the new design